core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.ts

```diff
@@ -578,25 +590,63 @@ export class DotPageStore extends ComponentStore<DotPagesState> {
         });
     };
 
-    private getFavoritePagesData = (limit: number, identifier?: string) => {
-        const identifierQuery = identifier ? `+identifier:${identifier}` : '';
+    private getFavoritePagesData = (limit: number, identifier?: string, url?: string) => {
```

Comment:
Maybe the parameters of this function could be one object

Reply:
Totally.

Reply:
done!